sdk/typescript/src/cryptography/ed25519-keypair.ts

```diff
@@ -69,18 +79,12 @@ export class Ed25519Keypair implements Keypair {
     options?: { skipValidation?: boolean },
   ): Ed25519Keypair {
     const secretKeyLength = secretKey.length;
-    if (secretKeyLength !== 64) {
-      // Many users actually wanted to invoke fromSeed(seed: Uint8Array), especially when reading from keystore.
-      if (secretKeyLength === 32) {
-        throw new Error(
-          'Wrong secretKey size. Expected 64 bytes, got 32. Similar function exists: fromSeed(seed: Uint8Array)',
-        );
-      }
+    if (secretKeyLength !== 32) {
```

Comment:
ditto re const or docs (generally avoid non defined numbers inside functions for readability)